packages/cli/tasks/accept_ownership.js

```diff
@@ -0,0 +1,18 @@
+/*global task*/
+/*eslint no-undef: "error"*/
+/*eslint no-unused-vars: ["error", { "varsIgnorePattern": "[NAME,DESC]" }]*/
+
+const {
+  TASK_INTERACT_NAME,
```

Comment:
I don't think that every task needs to know about every other task, so just import what's needed?

Reply:
I removed not implemented tasks and the imports from the implemented ones